Library/Homebrew/cmd/update-report.rb

```diff
@@ -136,6 +140,21 @@ def update_report
     Commands.rebuild_commands_completion_list
     link_completions_manpages_and_docs
     Tap.each(&:link_completions_and_manpages)
+
+    return if new_repository_version.blank?
+
+    ohai "Homebrew was updated to version #{new_repository_version}"
```

Comment:
Will this not be printed every time for people on the `stable` branch? This will also not be printed for people on the `master` branch if they are one commit past the tag.

Reply:
The `return if new_repository_version.blank?` on line 144 means that this will only run when users have updated to a new tag. So running `brew update && brew update` should only display the message the first time.

Yes, those on the `master` branch will only get this message if they happen to update when the latest tag is at `origin/HEAD`. I figured this was okay because the only ones on the `master` branch are developers. That's my justification for not putting the extra work into making it work for developers as well.

Reply:
I think it might be better (for both cases) to store in `.gitconfig` the last tag that we've displayed this message for so we can display it once per tag, even if the current commit isn't exactly on it. Thoughts?

Reply:
Makes sense to me. Do we have tooling for writing to `.gitconfig`? Or is it just `git -C $HOMEBREW_REPOSITORY config --local  --get lates-tag`?